mozilla-central/setup

```diff
@@ -77,135 +77,140 @@ INDEXED_GIT_REV=$(awk -v REV=$INDEXED_HG_REV '$2 ~ REV { print $1 }' git_hg.map)
 # so that we have analysis data for a version of the source we have.
 # Whether or not we do the build is decided in the "build" script in the same folder
 # as this file, which detects this scenario by the absence of the
-# target.mozsearch-index.zip file.
+# linux64.mozsearch-index.zip file.
 
 if [ -n "$INDEXED_GIT_REV" ]; then
     pushd $GIT_ROOT
     # Since we're using a prebuilt index at a particular rev, sync our local repo to the same rev
     git checkout $INDEXED_GIT_REV
     popd
 
-    # Download the C++ analysis, Rust save-analysis files, and generated sources tarballs.
+    rm -rf analysis && mkdir -p analysis
+    rm -rf objdir && mkdir -p objdir
+    rm -f analysis-files.list analysis-dirs.list
 
-    rm -rf $INDEX_ROOT/analysis
-    rm -rf $INDEX_ROOT/objdir
-
-    rm -f target.mozsearch-index.zip
-    wget -nv https://index.taskcluster.net/v1/task/gecko.v2.$REVISION.firefox.linux64-searchfox-debug/artifacts/public/build/target.mozsearch-index.zip
-    mkdir -p $INDEX_ROOT/analysis
-    unzip -q target.mozsearch-index.zip -d $INDEX_ROOT/analysis
-
-    rm -f target.mozsearch-rust.zip
-    wget -nv https://index.taskcluster.net/v1/task/gecko.v2.$REVISION.firefox.linux64-searchfox-debug/artifacts/public/build/target.mozsearch-rust.zip
-    mkdir -p $INDEX_ROOT/objdir
-    unzip -q target.mozsearch-rust.zip -d $INDEX_ROOT/objdir
-
-    rm -f target.generated-files.tar.gz
-    wget -nv https://index.taskcluster.net/v1/task/gecko.v2.$REVISION.firefox.linux64-searchfox-debug/artifacts/public/build/target.generated-files.tar.gz
-    tar -x -z -C $INDEX_ROOT/objdir -f target.generated-files.tar.gz
-
-    # Special cases - buildid.h and mozilla-config.h show up in two places in a regular
-    # objdir and the analysis tarball will correspondingly also have two instances of
-    # the analysis file. The generated-files tarball only has one copy, so we manually
-    # make the other copy to make things match up. The best would be if the gecko build
-    # system didn't actually produce two copies of these files.
-    cp $INDEX_ROOT/objdir/buildid.h $INDEX_ROOT/objdir/dist/include/buildid.h
-    cp $INDEX_ROOT/objdir/mozilla-config.h $INDEX_ROOT/objdir/dist/include/mozilla-config.h
-
-    date
+    # Download the artifacts from taskcluster for each platform that we're indexing
+    for PLATFORM in linux64 macosx64; do
+        date
```

Comment:
Maybe print the platform, too?

Reply:
Updated patch to print the platform. I did it after the revision check, before downloading the artifacts.